diff --git a/src/Tempest/Framework/Testing/Http/HttpRouterTester.php b/src/Tempest/Framework/Testing/Http/HttpRouterTester.php
index 1df72b6c04efbbaef047542f518f1fa5a34ae29e..bd8bf3f8233c88b38708cec3efb969f12438fe0c 100644
--- a/src/Tempest/Framework/Testing/Http/HttpRouterTester.php
+++ b/src/Tempest/Framework/Testing/Http/HttpRouterTester.php
@@ -36 +36 @@ final class HttpRouterTester
-    private(set) ?ContentType $contentType = null;
+    public ?ContentType $contentType = null;
@@ -38 +38 @@ final class HttpRouterTester
-    private(set) bool $includeSecFetchHeaders = true;
+    public bool $includeSecFetchHeaders = true;
@@ -40 +40 @@ final class HttpRouterTester
-    private(set) bool $throwExceptions = false;
+    public bool $throwExceptions = false;
@@ -248 +248 @@ public function sendRequest(Request $request): TestResponseHelper
-            $response = $router->dispatch(map($request)->with(RequestToPsrRequestMapper::class)->do());
+            $response = $router->dispatch(map($request)->with(RequestToPsrRequestMapper::class)->execute());
@@ -251 +251 @@ public function sendRequest(Request $request): TestResponseHelper
-                $response = $router->dispatch(map($request)->with(RequestToPsrRequestMapper::class)->do());
+                $response = $router->dispatch(map($request)->with(RequestToPsrRequestMapper::class)->execute());
