diff --git a/src/Tempest/Framework/Testing/Http/TestResponseHelper.php b/src/Tempest/Framework/Testing/Http/TestResponseHelper.php
index b3e244d53bd3e484a3086f203cc661904fd2502c..6c75fb56b3d2e985cef48f94250f999e5ea547ce 100644
--- a/src/Tempest/Framework/Testing/Http/TestResponseHelper.php
+++ b/src/Tempest/Framework/Testing/Http/TestResponseHelper.php
@@ -37,2 +37,2 @@ public function __construct(
-        private(set) Response $response,
-        private(set) Request $request,
+        public Response $response,
+        public Request $request,
@@ -40 +40 @@ public function __construct(
-        private(set) ?Throwable $throwable = null,
+        public ?Throwable $throwable = null,
