diff --git a/packages/mail/src/Testing/TestingMailer.php b/packages/mail/src/Testing/TestingMailer.php
index af747760bb8866b1044c17aabadd6fa6ec2f6f74..1f6b7153fc3c9856cd9333fd2d1c21962a1f0689 100644
--- a/packages/mail/src/Testing/TestingMailer.php
+++ b/packages/mail/src/Testing/TestingMailer.php
@@ -26 +26 @@ final class TestingMailer implements Mailer
-    private(set) array $sent = [];
+    public array $sent = [];
@@ -33 +33 @@ final class TestingMailer implements Mailer
-    private(set) array $failed = [];
+    public array $failed = [];
@@ -38 +38 @@ final class TestingMailer implements Mailer
-    private(set) ?Throwable $exception = null;
+    public ?Throwable $exception = null;
