diff --git a/packages/idempotency/src/Middleware/IdempotentCommandMiddleware.php b/packages/idempotency/src/Middleware/IdempotentCommandMiddleware.php
index 689382acf3479ca467c701e26601b6a13b706a69..c1dcf1d99484dd8b7acdc8973df12193444bbb2e 100644
--- a/packages/idempotency/src/Middleware/IdempotentCommandMiddleware.php
+++ b/packages/idempotency/src/Middleware/IdempotentCommandMiddleware.php
@@ -150 +150 @@ private function resolveOptions(object $command): ?array
-            $instance = ($this->commandBusConfig->handlers[$command::class] ?? null)
+            $instance = ($this->commandBusConfig->handlers[get_class($command)] ?? null)
@@ -184 +184 @@ private function resolveScope(object $command): string
-        return 'command:' . $command::class;
+        return 'command:' . get_class($command);
