diff --git a/packages/database/src/Exceptions/ModelDidNotHavePrimaryColumn.php b/packages/database/src/Exceptions/ModelDidNotHavePrimaryColumn.php
index e311557f60c71409fa0cfc5534b96dadfdc19569..e60b2890a48b907e84ef17b1f72e72f5c5ed204b 100644
--- a/packages/database/src/Exceptions/ModelDidNotHavePrimaryColumn.php
+++ b/packages/database/src/Exceptions/ModelDidNotHavePrimaryColumn.php
@@ -14 +14 @@ public static function neededForMethod(string|object $model, string $method): se
-            $model = $model::class;
+            $model = get_class($model);
@@ -23 +23 @@ public static function neededForRelation(string|object $model, string $relationT
-            $model = $model::class;
+            $model = get_class($model);
