diff --git a/packages/http/src/IsRequest.php b/packages/http/src/IsRequest.php
index 53408abb377a04527c713d307b9af0e46fe71409..144181e4e6518fb77ef3d998ca633623848b0c7d 100644
--- a/packages/http/src/IsRequest.php
+++ b/packages/http/src/IsRequest.php
@@ -20 +20 @@ trait IsRequest
-    private(set) Method $method;
+    public Method $method;
@@ -23 +23 @@ trait IsRequest
-    private(set) string $uri;
+    public string $uri;
@@ -26 +26 @@ trait IsRequest
-    private(set) ?string $raw = null;
+    public ?string $raw = null;
@@ -29 +29 @@ trait IsRequest
-    private(set) array $body = [];
+    public array $body = [];
@@ -32 +32 @@ trait IsRequest
-    private(set) RequestHeaders $headers;
+    public RequestHeaders $headers;
@@ -35 +35 @@ trait IsRequest
-    private(set) string $path;
+    public string $path;
@@ -38 +38 @@ trait IsRequest
-    private(set) array $query;
+    public array $query;
@@ -42 +42 @@ trait IsRequest
-    private(set) array $files;
+    public array $files;
