diff --git a/packages/http/src/Status.php b/packages/http/src/Status.php
index ae4feedb66a4f721b552768a5b16d254986998ff..5a1efcf57602be04c856e94ef767747355c35f02 100644
--- a/packages/http/src/Status.php
+++ b/packages/http/src/Status.php
@@ -7,2 +6,0 @@
-use ValueError;
-
@@ -12 +10 @@ enum Status: int
-    case CONTINUE = 100;
+    case HTTP_CONTINUE = 100;
@@ -86,11 +84 @@ public static function fromCode(int $code): self
-        try {
-            return self::from($code);
-        } catch (ValueError) {
-            return match (intdiv($code, 100) * 100) {
-                100 => self::CONTINUE,
-                200 => self::OK,
-                300 => self::MULTIPLE_CHOICES,
-                400 => self::BAD_REQUEST,
-                default => self::INTERNAL_SERVER_ERROR,
-            };
-        }
+        return self::from($code);
