diff --git a/packages/discovery/src/DiscoveryCache.php b/packages/discovery/src/DiscoveryCache.php
index 9f7873bd7fe0adc8ad00d3f70ff0e78206cbc299..514f294cb0ddd44661f9d10107054e5844c9ca07 100644
--- a/packages/discovery/src/DiscoveryCache.php
+++ b/packages/discovery/src/DiscoveryCache.php
@@ -26 +26 @@ public function __construct(
-        private(set) readonly DiscoveryCacheStrategy $strategy,
+        public readonly DiscoveryCacheStrategy $strategy,
@@ -36,3 +36 @@ public function withStrategy(DiscoveryCacheStrategy $strategy): self
-        return clone($this, [
-            'strategy' => $strategy,
-        ]);
+        return new self($strategy, $this->pool);
