diff --git a/packages/intl/src/MessageFormat/Formatter/MessageFormatter.php b/packages/intl/src/MessageFormat/Formatter/MessageFormatter.php
index fea3ed83b04ebd6d7c2459fd6e12c1bf437f46cb..eb5eea4d962b54a16fcebb69ac10c2e787f111e0 100644
--- a/packages/intl/src/MessageFormat/Formatter/MessageFormatter.php
+++ b/packages/intl/src/MessageFormat/Formatter/MessageFormatter.php
@@ -135 +135 @@ private function formatMessage(MessageNode $message): string
-        throw new FormattingException('Unknown message type: ' . $message::class);
+        throw new FormattingException('Unknown message type: ' . get_class($message));
@@ -152 +152 @@ private function formatComplexBody(ComplexBody $body): string
-        throw new FormattingException('Unknown complex body type: ' . $body::class);
+        throw new FormattingException('Unknown complex body type: ' . get_class($body));
@@ -263 +263 @@ private function formatPlaceholder(Placeholder $placeholder): string
-        throw new FormattingException('Unknown placeholder type: ' . $placeholder::class);
+        throw new FormattingException('Unknown placeholder type: ' . get_class($placeholder));
