diff --git a/packages/view/src/Parser/Token.php b/packages/view/src/Parser/Token.php
index 1ffe42d13a8eeca1699fa57f1fcb430baa99953b..0d262df83f6d7b0188f9a5bff7a360b24aad6bdd 100644
--- a/packages/view/src/Parser/Token.php
+++ b/packages/view/src/Parser/Token.php
@@ -12 +12 @@ final class Token
-    private(set) array $children = [];
+    public array $children = [];
@@ -14 +14 @@ final class Token
-    private(set) ?Token $parent = null;
+    public ?Token $parent = null;
@@ -16 +16 @@ final class Token
-    private(set) ?Token $endingToken = null;
+    public ?Token $endingToken = null;
@@ -18 +18 @@ final class Token
-    private(set) ?Token $closingToken = null;
+    public ?Token $closingToken = null;
@@ -21 +21 @@ final class Token
-    private(set) array $rawAttributes = [];
+    public array $rawAttributes = [];
@@ -23 +23 @@ final class Token
-    private(set) array $phpAttributes = [];
+    public array $phpAttributes = [];
@@ -25 +25 @@ final class Token
-    private(set) array $htmlAttributes = [];
+    public array $htmlAttributes = [];
@@ -27 +27 @@ final class Token
-    private(set) ?string $tag = null;
+    public ?string $tag = null;
