diff --git a/packages/validation/src/Exceptions/ValidationFailed.php b/packages/validation/src/Exceptions/ValidationFailed.php
index 33a8fe00375ebe9b0528914429f4ad998a7a9a70..395576abc9ca00bb12a0854c92058440f94f377e 100644
--- a/packages/validation/src/Exceptions/ValidationFailed.php
+++ b/packages/validation/src/Exceptions/ValidationFailed.php
@@ -22,4 +22,4 @@ public function __construct(
-        private(set) array $failingRules,
-        private(set) object|string|null $subject = null,
-        private(set) array $errorMessages = [],
-        private(set) ?string $targetClass = null,
+        public array $failingRules,
+        public object|string|null $subject = null,
+        public array $errorMessages = [],
+        public ?string $targetClass = null,
