diff --git a/packages/mapper/src/Mappers/ArrayToObjectMapper.php b/packages/mapper/src/Mappers/ArrayToObjectMapper.php
index 278a8f04fb2401d012d8e82fbaa93d252bc50d23..522a6157f13987d341d5b27c4c3672981a0524c0 100644
--- a/packages/mapper/src/Mappers/ArrayToObjectMapper.php
+++ b/packages/mapper/src/Mappers/ArrayToObjectMapper.php
@@ -170 +170 @@ private function setChildParentRelation(object $parent, mixed $child, ClassRefle
-            if ($childProperty->getType()->equals($parent::class)) {
+            if ($childProperty->getType()->equals(get_class($parent))) {
@@ -172 +172 @@ private function setChildParentRelation(object $parent, mixed $child, ClassRefle
-            } elseif ($childProperty->getIterableType()?->equals($parent::class)) {
+            } elseif ($childProperty->getIterableType()?->equals(get_class($parent))) {
@@ -210 +210 @@ public function resolveValue(PropertyReflector $property, mixed $value): mixed
-        if (is_object($value) && $property->getType()->matches($value::class)) {
+        if (is_object($value) && $property->getType()->matches(get_class($value))) {
