diff --git a/packages/validation/src/Validator.php b/packages/validation/src/Validator.php
index 9ae3e1bbdb95546b14c619ed2e3f8e0530daebff..4b2095a03c2ca131cf0705d565a43b9e21295b17 100644
--- a/packages/validation/src/Validator.php
+++ b/packages/validation/src/Validator.php
@@ -60 +60 @@ public function validateObject(object $object): void
-            throw $this->createValidationFailureException($failingRules, $object, $object::class);
+            throw $this->createValidationFailureException($failingRules, $object, get_class($object));
@@ -265 +265 @@ private function getTranslationKey(Rule|FailingRule $rule): string
-        return str($rule::class)
+        return str(get_class($rule))
