diff --git a/vendor/symfony/uid/Uuid.php b/vendor/symfony/uid/Uuid.php
--- a/vendor/symfony/uid/Uuid.php
+++ b/vendor/symfony/uid/Uuid.php
@@ -66 +66 @@ class Uuid extends AbstractUid
-        if (self::MAX === $uuid = strtr($uuid, 'F', 'f')) {
+        if (self::MAX === ($uuid = strtr($uuid, 'F', 'f'))) {
@@ -86 +86 @@ class Uuid extends AbstractUid
-    final public static function v1(): UuidV1
+    public static function v1(): UuidV1
@@ -91 +91 @@ class Uuid extends AbstractUid
-    final public static function v3(self $namespace, string $name): UuidV3
+    public static function v3(self $namespace, string $name): UuidV3
@@ -99 +99 @@ class Uuid extends AbstractUid
-    final public static function v4(): UuidV4
+    public static function v4(): UuidV4
@@ -104 +104 @@ class Uuid extends AbstractUid
-    final public static function v5(self $namespace, string $name): UuidV5
+    public static function v5(self $namespace, string $name): UuidV5
@@ -112 +112 @@ class Uuid extends AbstractUid
-    final public static function v6(): UuidV6
+    public static function v6(): UuidV6
@@ -117 +117 @@ class Uuid extends AbstractUid
-    final public static function v7(): UuidV7
+    public static function v7(): UuidV7
@@ -122 +122 @@ class Uuid extends AbstractUid
-    final public static function v8(string $uuid): UuidV8
+    public static function v8(string $uuid): UuidV8
@@ -138 +138 @@ class Uuid extends AbstractUid
-        if (false === $uuid = self::transformToRfc9562($uuid, $format)) {
+        if (false === ($uuid = self::transformToRfc9562($uuid, $format))) {
@@ -176 +176 @@ class Uuid extends AbstractUid
-        if (false !== $cmp = uuid_compare($this->uid, $other->uid)) {
+        if (false !== ($cmp = uuid_compare($this->uid, $other->uid))) {
