diff --git a/packages/upgrade/tests/RectorTester.php b/packages/upgrade/tests/RectorTester.php
index f3aa25ed1bc864e6b082c0b793d1e6f921a0f746..59f47bc952c9fc51342ede8f397a35210d896bd6 100644
--- a/packages/upgrade/tests/RectorTester.php
+++ b/packages/upgrade/tests/RectorTester.php
@@ -14 +14 @@ final class RectorTester
-    private(set) string $fixturePath;
+    public string $fixturePath;
@@ -16 +16 @@ final class RectorTester
-    private(set) string $actual;
+    public string $actual;
