diff --git a/packages/mapper/src/CasterFactory.php b/packages/mapper/src/CasterFactory.php
index 43fcd6268cbd5502dcd19c27a26ef1be8e18b246..46847aebb1132f1f77ed8e9d26959ea74d159e83 100644
--- a/packages/mapper/src/CasterFactory.php
+++ b/packages/mapper/src/CasterFactory.php
@@ -22 +22 @@ final class CasterFactory
-    private(set) array $casters = [];
+    public array $casters = [];
@@ -24 +24 @@ final class CasterFactory
-    private(set) Context|UnitEnum|string|null $context = null;
+    public Context|UnitEnum|string|null $context = null;
