diff --git a/packages/mapper/src/ObjectFactory.php b/packages/mapper/src/ObjectFactory.php
index b0f71f5d62f3b9129178a24cb09c0b877b401275..1e03e0a719e4432e585d27bc908710187cf2e1ff 100644
--- a/packages/mapper/src/ObjectFactory.php
+++ b/packages/mapper/src/ObjectFactory.php
@@ -115,3 +115,2 @@ public function in(Context|UnitEnum|string|null $context): self
-        $clone = clone($this, [
-            'context' => $context,
-        ]);
+        $clone = clone $this;
+        $clone->context = $context;
