diff --git a/packages/auth/src/OAuth/Testing/TestingOAuthClient.php b/packages/auth/src/OAuth/Testing/TestingOAuthClient.php
index 5a791ea05e06be83534a1c2bb2d2bd9b9c3310f5..98d4f85ff2fa07d0123853652f3d2490521e138e 100644
--- a/packages/auth/src/OAuth/Testing/TestingOAuthClient.php
+++ b/packages/auth/src/OAuth/Testing/TestingOAuthClient.php
@@ -28 +28 @@ final class TestingOAuthClient implements OAuthClient
-    private(set) ?string $baseUrl = null;
+    public ?string $baseUrl = null;
@@ -61,2 +61,2 @@ public function __construct(
-        private(set) OAuthUser $user,
-        private(set) readonly OAuthConfig $config,
+        public OAuthUser $user,
+        public readonly OAuthConfig $config,
