diff --git a/packages/log/src/Config/DailyLogConfig.php b/packages/log/src/Config/DailyLogConfig.php
index 69c394ed87eed1ddbebb4c8d8caa122c6dfdacf2..c4cd7f374f1a0da525f149c86182d560594196a0 100644
--- a/packages/log/src/Config/DailyLogConfig.php
+++ b/packages/log/src/Config/DailyLogConfig.php
@@ -39,8 +39,8 @@ public function __construct(
-        private(set) string $path,
-        private(set) int $maxFiles = 31,
-        private(set) LogLevel $minimumLogLevel = LogLevel::DEBUG,
-        private(set) array $channels = [],
-        private(set) bool $lockFilesDuringWrites = false,
-        private(set) ?int $filePermission = null,
-        private(set) ?string $prefix = null,
-        private(set) UnitEnum|string|null $tag = null,
+        public string $path,
+        public int $maxFiles = 31,
+        public LogLevel $minimumLogLevel = LogLevel::DEBUG,
+        public array $channels = [],
+        public bool $lockFilesDuringWrites = false,
+        public ?int $filePermission = null,
+        public ?string $prefix = null,
+        public UnitEnum|string|null $tag = null,
