diff --git a/packages/log/src/Config/WeeklyLogConfig.php b/packages/log/src/Config/WeeklyLogConfig.php
index 0df4f10e0e5868299971eb6eea86998d4abd2f83..1f4f2881988063b91f94ac249d003082c37b2f84 100644
--- a/packages/log/src/Config/WeeklyLogConfig.php
+++ b/packages/log/src/Config/WeeklyLogConfig.php
@@ -39,8 +39,8 @@ public function __construct(
-        private(set) string $path,
-        private(set) int $maxFiles = 5,
-        private(set) LogLevel $minimumLogLevel = LogLevel::DEBUG,
-        private(set) array $channels = [],
-        private(set) ?string $prefix = null,
-        private(set) bool $lockFilesDuringWrites = false,
-        private(set) ?int $filePermission = null,
-        private(set) UnitEnum|string|null $tag = null,
+        public string $path,
+        public int $maxFiles = 5,
+        public LogLevel $minimumLogLevel = LogLevel::DEBUG,
+        public array $channels = [],
+        public ?string $prefix = null,
+        public bool $lockFilesDuringWrites = false,
+        public ?int $filePermission = null,
+        public UnitEnum|string|null $tag = null,
